CMakeLists.txt

```diff
@@ -352,6 +352,34 @@ if (WIN32)
 	configure_file(properties.rc.cmakein ${LIBIIO_RC} @ONLY)
 endif()
 
+option(COVERITY_BUILD "Turn on things for Coverity Builds" OFF)
```

Comment:
Shouldn't we document this on https://github.com/analogdevicesinc/libiio/blob/master/README_BUILD.md?

Reply:
No. Normal users should not use this flag. It will generate a source tree which may not build.

Reply:
Fair point.